stl/inc/utility

```diff
@@ -644,6 +644,88 @@ template <size_t _Idx>
 inline constexpr in_place_index_t<_Idx> in_place_index{};
 #endif // _HAS_CXX17
 
+#if _HAS_CXX20
+template <class _Ty>
+inline constexpr bool _Is_integral_not_bool_char = is_integral_v<_Ty>

```

Comment:
Naming: We currently have `_Is_nonbool_integral`. I think that `not_bool_char`, in addition to being inconsistent with `nonbool`, is also somewhat confusing - this type trait rejects both `char` and `char16_t` etc. but permits `signed char` and `unsigned char`.

I recommend imitating the Standardese and calling this `_Is_standard_integer`. (We can ignore the "extended" bit since we don't have any such types.) Then this would be consistent with:
https://github.com/microsoft/STL/blob/43e07f3fad934b710d2de21ffdcf4cd72267c826/stl/inc/bit#L39-L41